query/templates/search/contentQuery.tmpl

```diff
@@ -1,11 +1,16 @@
 {
+explain: true,
 {{/* content query */}}
  {{- if .From -}}
  	"from" : {{- .From}},
  {{- end}}
  "size" : {{.Size}},
  "query" : {
      "bool" : {
+     "should": [
+             {{template "nlpCategory.tmpl" .}}
+        ],
+
```

Comment:
if the nlp is turned off and nothing is present here, will this present a problem?